kubescape-operator.yaml

```
@@ -0,0 +1,88 @@
+package:
+  name: kubescape-operator
+  version: "0.2.92"
+  epoch: 0
+  description: Kubescape-Operator is an in-cluster component of the Kubescape security platform.
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 81602c62580d67a78f30ff4670087392d39842c9
+      repository: https://github.com/kubescape/operator
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/go-viper/mapstructure/v2@v2.3.0
+        github.com/inspektor-gadget/inspektor-gadget@v0.40.0
+
+  - uses: go/build
+    with:
+      packages: .
+      output: operator
+
+update:
+  enabled: true
+  github:
+    identifier: kubescape/operator
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        operator --help
```

```suggestion
        set -o pipefail
        operator --help | grep -F -e "(something from the output)"
```
Whenever you are doing --help checks, please do this so you can make sure the output is not being garbled somehow (this happened on the QEMU package and we didnt catch it)

---

Hi @tulilirockz, thanks for the suggestion! I tried implementing it, but the test fails because the kubescape-operator has very limited help output.

Here's what it currently prints:
```
Usage of operator:
  -kubeconfig string
        Paths to a kubeconfig. Only required if out-of-cluster.
```
since --help only lists this one flag, and the output is minimal, the grep works correctly but the test still exits with status 1. So for now, I’m keeping the original check to avoid unnecessary test failures.